Github/server.js

```diff
@@ -47,12 +49,32 @@ app.get('/', (req, res) => {
             res.status(err.statusCode).send(err.statusMessage);
           });
       })
-      .catch(err => console.log(err));
+      .catch(err => {
+        res.status(err.statusCode).send(err.statusMessage);
+      });
   } else {
     res.json(storedData);
   }
 });
 
+app.get('/helpWanted', (req, res) => {
```

Comment:
Prefer hyphens over camelcase in urls. I believe this is also the accepted standard for in REST APIs. 
```suggestion
app.get('/help-wanted', (req, res) => {
```